### ISSUE TITLE
Connection error occurred Notice: Array to string conversion

### ISSUE THREAD

**Issue by KrissMcCone** [GeniusesOfSymfony/WebSocketBundle#15]
Hey. It's me again, i'm really sorry. :)

After update, all my code doesn't work. I reseted all to code from tutorial, I mean js and Topic class.

But i still have the same error and i don't know what to do.

Connection error occurred Notice: Array to string conversion in /vendor/gos/web-socket-bundle/Server/App/Registry/TopicRegistry.php line 46 {"connection_id":657,"session_id":"553cb0a9eae46","client":"anon-553cb0a9eae46"}

**Comment by KrissMcCone** [comment]
Oh, Symfony don't map my service.yml inside Bundle. I have to fix it, or registry topic in main service. It will be nice to add exepction that no one topic is loaded instead of this error. :)

**Comment by ProPheT777** [comment]
Well it's my fault, we throw an exception https://github.com/GeniusesOfSymfony/WebSocketBundle/blob/master/Server/App/Registry/TopicRegistry.php#L42 but I made a genius thing ... 

My message exception create another exception

```
            throw new \Exception(sprintf(
                'Topic %s does\'nt exist in [ %s ]',
                $topicName,
                array_keys($this->topics)
            ));
```

Should be `implode(', ', array_keys($this->topics)); `

I will fix it

**Comment by ProPheT777** [comment]
Just released v1.0.5 who fix that !
